src/components/BookList.jsx

```diff
@@ -0,0 +1,32 @@
+function BookList() {
+  const books = [{
+    title: 'pride and prejudice',
+    author: 'jane austen',
+  }, {
+    title: 'unleash man within',
+    author: 'sathiya sam',
+  }, {
+    title: 'Man with a plan',
+    author: 'Matt Le Blanc',
+  }, {
+    title: 'Breaking Bad',
+    author: 'Vince Gilligan',
+  }];
+
+  return (
+    <div className="bookList container">
+      {books.map((book) => (
+        <div key="book">
+          <h3>{book.title}</h3>
+          <p>{book.author}</p>
+          <div style={{ display: 'flex', gap: '20px' }}>
+            <button type="button">Comment</button>
+            <button type="button">Remove</button>
+            <button type="button">Edit</button>
+          </div>
+        </div>
```

Comment:
[ OPTIONAL CHANGE ] But Highly I highly recommend you implement it.
The previous reviewer did not mention it and so I'm marking it as optional but I highly recommend you work on it because the way I'm going to ask you to implement it, is the required way asked in the project.👍🏽

- [ ] Kindly move this highlighted code into a separate component called Book.js, then when you loop over the books as you did here, you pass the looped data as a prop to that Book component that you will be imported here. Let's build our apps with small pieces of components and pass props of data to them so some components look lean and clean. Kindly work it out like that 👍🏽